compiler/rustc_parse/src/parser/stmt.rs

```diff
@@ -247,6 +247,21 @@ impl<'a> Parser<'a> {
     /// Parses a local variable declaration.
     fn parse_local(&mut self, attrs: AttrVec) -> PResult<'a, P<Local>> {
         let lo = self.prev_token.span;
+
+        if self.eat_keyword(kw::Const) {
```

Comment:
`let const { expression } = ...` must be accepted. The `const {}` there is an inline const pattern.

Reply:
Fixed!